.codeclimate.yml

```diff
@@ -30,15 +30,20 @@ engines:
     enabled: true
   fixme:
     enabled: true
+    checks:
+      XXX:
+        enabled: false
```

Comment:
For some reason Rubocop doesn't like the 'XXX-XXX-XXXX' placeholders that we discussed so I'm disabling that check.